tests/src/testModels.js

```diff
@@ -302,6 +305,7 @@ describe('Model', function() {
         expect(permissionsDefault.devices_input).toBe(true);
         expect(permissionsDefault.devices_kvm).toBe(true);
         expect(permissionsDefault.devices_shm).toBe(true);
+        expect(permissionsDefault.devices_usb).toBe(true);
         expect(permissionsDefault.devices_all).toBe(true);
         expect(permissionsDefault.features_bluetooth).toBe(true);
         expect(permissionsDefault.features_devel).toBe(true);
```

Comment:
This is failing because there are a few tests that "count" the number the "active" permissions (depending on different Flatpak versions), so you need to increase the total number of permissions (in this same file) and also increase that number in the tests where the version of Flatpak is "very new".